site/cypress/e2e/Cypress-Feature/grade_inquiries.spec.js

```diff
@@ -9,18 +11,27 @@ describe('Test cases revolving around grade inquiries', () => {
         cy.get('[data-testid="grade-inquiry-due-date"]').clear();
         cy.get('[data-testid="grade-inquiry-due-date"]').type(date);
         cy.get('[data-testid="grade-inquiry-due-date"]').type('{enter}');
+        cy.get('[data-testid="save-status"]', { timeout: 10000 }).should('have.text', 'All Changes Saved');
     };
 
     it('should test normal submission grade inquiry panel', () => {
-        cy.login();
-        const gradeableId = 'grades_released_homework';
         const gradeInquiryDeadlineDate = '9998-01-01 00:00:00';
```

Comment:
Can we move all of the constants to the top of the file to be more consistant with other files? (Even if you didn't add them I think it'd be a good idea.

Reply:
👍